TestingExamples/LoadGenerator/Services/LoadGenerator.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace LoadGenerator.Services;
+
+public class LoadGenerator
+{
+    private DateTime _startTime;
+    private DateTime _endTime;
+    private long _totalLoadGenerated;
+
+    public LoadGenerator()
```

Comment:
Is this needed?

Reply:
Not currently, no. It was there for me to build out further but was un-implemented in the current state I can remove it for now though and always add it in later when I go to flesh this out more in the future